### PR TITLE
DAOS-7174 doc: remove duplicated sections

### PR DESCRIPTION
**Pull request by johannlombardi**
- merge admin/installation section into the QSG
- merge DAOS from scratch instructions from admin guide into QSG
- move the newly created DAOS in Docker instructions to the QSG
- move DFuse pool and system modes to specific advanced usage
  section to avoid confusion
- point at install scripts to install prerequesites when building
  from scratch

doc-only: true

Signed-off-by: Johann Lombardi <johann.lombardi@intel.com>

### CODE REVIEW

**Comment by knard-intel** [comment]
The `utils/scripts/*.sh`  bash script are not available with your PR. Moreover, they are not executable on the master branch. They are also not present in the release/2.0 branch. Maybe it is planned to backport with another PR ?

File: docs/QSG/build_from_scratch.md [hunk 1]
```diff
@@ -2,296 +2,103 @@
...
-
-		python3-tabulate \\
+### CentOS
 
-		scons \\
+On CentOS7, please run the following command from the DAOS tree.
 
-		sg3\_utils \\
+```bash
+$ ./utils/scripts/install-centos7.sh
```
Comment (knard-intel) [inline docs/QSG/build_from_scratch.md:44]: Should be called with sudo or the commands inside the bash script.

File: docs/user/filesystem.md [hunk 2]
```diff
@@ -502,3 +439,59 @@ $ D_IL_REPORT=-1 LD_PRELOAD=/usr/lib64/libioil.so dd if=/dev/zero of=./bar bs=1G
...
+  Container Label: mycont2
+  Container Type : POSIX
+
+Successfully created container 0db21789-5372-4f2a-b7bc-14c0a5e968df
+$ dfuse -m /tmp/dfuse --pool tank
+$ ls -l /tmp/dfuse/
+ls: cannot open directory '/tmp/dfuse/': Operation not supported
+$ ls -l /tmp/dfuse/0db21789-5372-4f2a-b7bc-14c0a5e968df
+total 0
+$ ls -l /tmp/dfuse/8a8f08bb-5034-41e8-b7ae-0cdce347c558
```
Comment (knard-intel) [inline docs/user/filesystem.md:468]: Creation of this pool seems to be missing in this new documentation.

File: mkdocs.yml [hunk 3]
```diff
@@ -71,19 +71,18 @@ nav:
     - Installation and Setup:
         - 'CentOS': 'QSG/setup_centos.md'
         - 'openSUSE': 'QSG/setup_suse.md'
+	- 'DAOS in Docker': 'QSG/docker.md'
```
Comment (Michael-Hennecke) [inline mkdocs.yml:74]: tab/space issue?

File: docs/QSG/build_from_scratch.md (hunk 1 above)
Comment (johannlombardi) [reply docs/QSG/build_from_scratch.md:48]: Fixed in new version.

File: docs/user/filesystem.md (hunk 2 above)
Comment (johannlombardi) [reply docs/user/filesystem.md:478]: Fixed in new version.

File: mkdocs.yml (hunk 3 above)
Comment (johannlombardi) [reply mkdocs.yml]: Fixed in new version.

**Review by Michael-Hennecke — DISMISSED** [review]

**Review by Michael-Hennecke — APPROVED** [review]

**Review by knard-intel — APPROVED** [review]
OK from my side.

**Pull request merged**